modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -188,13 +188,12 @@ the assumption is that the `airline` collection exists within a scope `us`.
 
 [source,golang]
 ----
-myCollection := "airline"
-query := fmt.Sprintf("SELECT x.* FROM %s x LIMIT 10;", myCollection)
-        rows, err := us.Query(query, &gocb.QueryOptions{})
-        // check query was successful
-        if err != nil {
-                panic(err)
-        }
+scope := cluster.Bucket("mybucket").Scope("myscope")
```

Comment:
Ah, I just spotted line 187 above, which suggests the scope is called `us`. That's a terrible name for a scope though, so perhaps better to change that line?